.github/workflows/aspect-workflows-template.yaml

```diff
@@ -0,0 +1,142 @@
+# This reusable GitHub Actions workflows must either be vendored into the target repository as is
+# the case here -or- a fork made of https://github.com/aspect-build/workflows-action repository and
+# the reusable workflow referenced from the fork within the same organization as the target
+# repository.
+#
+# At this time, GitHub Actions does not allow referencing reusable Workflows from public
+# repositories in other organizations. See
```

Comment:
we are in the same organization, so you shouldn't need to vendor this file

Reply:
I know your goal is to make this an example, but it's an example for sales calls, not for committed trial customers who are happy to have our guidance. So I think we ought to make this simple where possible.

Reply:
I think these will serve as full fidelity examples for the docsite and for trial users who want to be more self-serve... but it is also a lot of code to vendor and maintain when we don't need to. Perhaps we use write_source_file to keep it up to date?

Reply:
My guess is that most trial users would prefer to vendor with a write_source_files vs fork the repo since forking requires another set of hoops to jump through with whoever is managing the GitHub account. The fork is also another place to have to sync whereas an http_archive to a particular version with a write_source_file is all in the same repo.

Reply:
As the one datapoint here, I discussed with AAI eng leadership and they decided forking the repo is better, but I don't think we had a good write_source_files proposal for them at the time.

I don't have a strong opinion here, and it's perhaps "cheating" a bit to leave this file out just because the repo is under the aspect-build org so we can be "special"

Reply:
Ok, I'll go with write_source_files and mention at the use site that it can be forked instead.

Reply:
Guess I can point to the AAI repo as an example of that since its public.